pkg/storage/client_raft_test.go

```diff
@@ -2900,9 +2904,15 @@ func TestStoreRangeRemoveDead(t *testing.T) {
 		return errors.Errorf("expected %d replicas; have %+v", zone.NumReplicas, replicas)
 	})
 
-	// Set the dead store and wait for the replica on the dead node to
-	// be removed and the replacement added. Use a store other than the leader.
-	atomic.StoreInt32(&deadStoreID, int32(getRangeMetadata(roachpb.RKeyMin, mtc, t).Replicas[1].StoreID))
+	// Stop a store which will be rebalanced away from. We can't use the very first
+	// one since getRangeMetadata is hard-coded to query that one, so we'll use the
+	// one after that.
+	storeID := mtc.stores[1].StoreID() // 2
+	atomic.StoreInt32(&deadStoreID, int32(storeID))
```

Comment:
No need to keep `deadStoreID` anymore. Remove this setting here and remove the variable declaration above.

Reply:
You also don't need `storeID`.

Reply:
Done, thanks!